frontend/src/app/modules/main/create-edit-playlist/edit-playlist-modal/edit-playlist-modal.component.ts

```diff
@@ -0,0 +1,116 @@
+import {
+  Component, EventEmitter, Input, OnInit, Output
+} from '@angular/core';
+import {
+  FormControl, FormGroup, Validators
+} from '@angular/forms';
+import { AccessType } from 'src/app/models/playlist/accessType';
+import { EditedPlaylist } from 'src/app/models/playlist/editedPlaylist';
+
+@Component({
+  selector: 'app-edit-playlist-modal',
+  templateUrl: './edit-playlist-modal.component.html',
+  styleUrls: ['./edit-playlist-modal.component.sass']
+})
+export class EditPlaylistModalComponent implements OnInit {
+  file: File;
+  public editPlaylistForm!: FormGroup;
+  public selectControlValues: AccessType[] = [AccessType.secret, AccessType.collaborative, AccessType.default];
+
+  public tempIconURL: string;
+
+  @Input() editedPlaylist: EditedPlaylist;
+
+  @Output()
+  isClosed = new EventEmitter<void>();
+
+  @Output()
+  editPlaylist = new EventEmitter<EditedPlaylist>();
+
+  ngOnInit() {
+    this.editPlaylistForm = new FormGroup({
+      name: new FormControl(
+        this.editedPlaylist.name,
+        [
+          Validators.required,
+          Validators.maxLength(50)
+        ]
+      ),
+      description: new FormControl(
+        this.editedPlaylist.description,
+        [
+          Validators.maxLength(200)
+        ]
+      ),
+      iconURL: new FormControl(''),
+      accessType: new FormControl(
+        this.editedPlaylist.accessType,
+        [Validators.required]
+      )
+    });
+
+    this.tempIconURL = this.editedPlaylist.iconURL;
+  }
+
+  get name() {
+    return this.editPlaylistForm.get('name')!;
+  }
+
+  get description() {
+    return this.editPlaylistForm.get('description')!;
+  }
+
+  get iconURL() {
+    return this.editPlaylistForm.get('iconURL')!;
+  }
+
+  get accessType() {
+    return this.editPlaylistForm.get('accessType')!;
+  }
+
```

Comment:
We could get rid of so many getters (here we have just 4 fields, but imagine if we had 20 fields). Use ngModel
https://angular.io/guide/forms